README.md

```diff
@@ -17,3 +17,11 @@ Open [http://localhost:3000](http://localhost:3000) with your browser to see the
 ## Deploy
 
 The app is based on [Next.JS](https://nextjs.org/) and is automatically built & deployed to GitHub Pages when pushing to the `main` branch.
+
+## Internal Server Configuration
+
+This app supports overriding of the default rekor server instance for those running private instances of the the sigstore stack.
+Create a `.env.local` file at the root and include in it this environment variable
+```properties
+NEXT_PUBLIC_REKOR_DEFAULT_DOMAIN=https://privaterekor.sigstore.dev
```

Comment:
Given that this would be for private instances, it feels like we should not use `PUBLIC` in the variable name?

Reply:
Good point! I think however this might be a requirement defined by Next.js. Otherwise the variable would not be available for use in the browser.

> Non-NEXT_PUBLIC_ environment variables are only available in the Node.js environment, meaning they aren't accessible to the browser (the client runs in a different environment).
> In order to make the value of an environment variable accessible in the browser, Next.js can "inline" a value, at build time, into the js bundle that is delivered to the client, replacing all references to process.env.[variable] with a hard-coded value. To tell it to do this, you just have to prefix the variable with NEXT_PUBLIC_. For example:

Reply:
perhaps it's worth adding some additional comments in the tsx files to denote the purpose? It's pretty confusing, though now I see it wasn't intended

Reply:
> perhaps it's worth adding some additional comments in the tsx files to denote the purpose? It's pretty confusing, though now I see it wasn't intended

I'm on it sir!

Reply:
Added a comment in the context.tsx file. Let me know if that's sufficient